### PR TITLE
[Enhancement] Ignore the property "replication_num" in lake table

### PR DESCRIPTION
**Pull request by sduzh**
Signed-off-by: sduzh <zhuming9011@qq.com>

## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 43 / 47 (91.49%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/LakeTable.java|0|1|00.00%|[174]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|41|44|93.18%|[2094, 2384, 2474]|
|:large_blue_circle:|com/starrocks/alter/LakeTableAlterJobV2Builder.java|2|2|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java [hunk 1]
```diff
@@ -2742,18 +2739,16 @@ public void replayCreateMaterializedView(String dbName, MaterializedView materia
...
-        Preconditions.checkArgument(replicationNum > 0);
-
         DistributionInfo.DistributionInfoType distributionInfoType = distributionInfo.getType();
         if (distributionInfoType != DistributionInfo.DistributionInfoType.HASH) {
             throw new DdlException("Unknown distribution type: " + distributionInfoType);
         }
 
         int bucketNum = distributionInfo.getBucketNum();
-        List<Long> shardIds = stateMgr.getStarOSAgent().createShards(bucketNum, replicationNum,
+        List<Long> shardIds = stateMgr.getStarOSAgent().createShards(bucketNum, 1,
```
[the remark this answers is not in the document]
Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java]: Updated, PTAL

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 45 / 49 (91.84%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/LakeTable.java|0|1|00.00%|[174]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|41|44|93.18%|[2094, 2384, 2474]|
|:large_blue_circle:|com/starrocks/alter/LakeTableAlterJobV2Builder.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/lake/StarOSAgent.java|2|2|100.00%|[]|

**Review by wyb — APPROVED** [review]

**Review by imay — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Pull request merged**